config/mocks/wallet-options-v4.json

```diff
@@ -50,6 +50,25 @@
           "showNewTagSidenav": true,
           "txListAppRoute": "/gbp/transactions"
         },
+        "USD": {
+          "availability": {
+            "send": false,
+            "request": false
+          },
+          "coinCode": "USD",
+          "coinTicker": "USD",
+          "colorCode": "fiat",
+          "displayName": "U.S Dollars",
+          "icons": {
+            "default": "usdt",
+            "circle": "usdt",
+            "circleFilled": "usdt"
```

Comment:
@milan-bc usdt placeholder for now please use usd and add the icon

Reply:
Done